### PR TITLE
1446 location movement sync

### LINKED ISSUE #1674: location_movement integration
Split out from #1446

### PR DESCRIPTION
**Pull request by clemens-msupply**
Part of #1446 
Closes #1674 

Implements location_movment sync. For testing run the tests/integration tests.

### CODE REVIEW

**Review by mark-prins — CHANGES_REQUESTED** [review]
🎉  working nicely! Just the version number bump for migrations - I had to run them manually to test, and couldn't figure out at first why it wasn't working!

File: server/service/src/sync/translations/location_movement.rs [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+        };
+
+        Ok(Some(vec![RemoteSyncRecordV5::new_upsert(
+            changelog,
+            LEGACY_TABLE_NAME,
+            serde_json::to_value(&legacy_row)?,
+        )]))
+    }
+
+    fn try_translate_push_delete(
```
Comment (andreievg) [inline server/service/src/sync/translations/location_movement.rs:132]: technically I don't think we will be deleting any location_movements

**Review by andreievg — APPROVED** [review]
Looks great thanks. just the note by @mark-prins about migration.

I think this great PR to have as an example of what is needed for a remote record to sync, I'll remember that for when we do docs/examples

File: server/service/src/sync/translations/location_movement.rs (hunk 1 above)
Comment (clemens-msupply) [reply server/service/src/sync/translations/location_movement.rs:132]: ok thanks, was briefly wondering about it, should I remove?

**Review by mark-prins — APPROVED** [review]
👍 Nice, thanks for changes!

File: server/service/src/sync/translations/location_movement.rs (hunk 1 above)
Comment (andreievg) [reply server/service/src/sync/translations/location_movement.rs:132]: Up to you really, it's not much code (although my preference would be to only have code that is used)

**Pull request merged**